packages/copyable/package.json

```diff
@@ -0,0 +1,24 @@
+
+  {
+    "name": "@leafygreen-ui/copyable",
+    "version": "0.9.0",
+    "description": "leafyGreen UI Kit Copyable",
+    "main": "./dist/index.bundle.js",
+    "types": "./dist/index.d.ts",
+    "scripts": {
+      "build": "../../node_modules/.bin/webpack --config ../../webpack.config.js"
+    },
+    "license": "Apache-2.0",
+    "publishConfig": {
+      "access": "public"
+    },
+    "dependencies": {
+      "@leafygreen-ui/button": "^7.0.0",
+      "@leafygreen-ui/emotion": "^2.0.2",
+      "@leafygreen-ui/icon": "^6.5.0",
+      "@leafygreen-ui/palette": "^2.0.2",
+      "@leafygreen-ui/tooltip": "^3.3.7",
+      "@leafygreen-ui/typography": "^4.2.2",
+      "clipboard": "^2.0.6"
```

Comment:
Wonder if we should set this to `^2.0.0` to have a bigger chance of having this package deduped from consuming applications. For example in Cloud we have this set to `2.0.6` so we're good, but if we had it set to `2.0.5` it would have resulted in a duped dep.

Reply:
I just chose the version that the `Code` component was using. I'd prefer to keep those in sync if possible, even if it means changing them to both to`^2.0.0`. though I don't quite understand why yarn/node wouldn't just use the `2.0.6` for the `2.0.5` dependency too.